packages/bootleg-app/src/DownloadCard.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { drizzleConnect } from 'drizzle-react';
+import { Heading, Card, Link, Text } from 'rimble-ui';
+
+const DownloadCard = ({ accounts, drizzzleStatus }) => {
+  return (
+    <Card>
+      <Heading.h2>
+        Congratulations! You are a franchisor and can access the bootleg video
+      </Heading.h2>
+      Download the video from this link:{' '}
```

Comment:
I prefer things like 

> Congratulations! You are a franchisor and can [download](https://s3.amazonaws.com/web3studio-bootlegs/outsideOUTSIDE-SXSW2019-Bootleg.mp4) the bootleg video

Rather than "Click this link"

Reply:
I was hoping to drop some kinda 
![image](https://user-images.githubusercontent.com/1624/57156099-26d77380-6dab-11e9-83c2-0138c71f4371.png) this on it... and make it a button.

Reply:
yikes! That was bigger than I expected.